.werft/vm/manifests.ts

```diff
@@ -135,3 +124,70 @@ type UserDataSecretManifestOptions = {
   namespace: string,
   secretName: string
 }
+
+// follow advice from https://github.com/gitpod-io/gitpod/pull/7272 to use a secret.
```

Comment:
I think we should just remove this comment - in the PR I switched from using a secret to just inlining the cloudinit, but I wasn't aware of the size restriction, so going back to a secret makes sense ☺️

Reply:
ok, removing.